examples/csharp/aws/cdktf.json

```diff
@@ -1,8 +1,11 @@
 {
   "language": "csharp",
   "app": "dotnet run -p MyTerraformStack.csproj",
-  "terraformProviders": ["aws@~> 2.0"],
+  "terraformProviders": [
+    "aws@~> 2.0"
+  ],
   "context": {
     "excludeStackIdFromLogicalIds": "true"
-  }
-}
+  },
+  "projectId": "87752e30-8860-4e59-98a4-86d2603d31fc"
```

Comment:
What happens if folks copy config from examples including the `projectId`?
I think it might make sense to have no `projectId` in our examples – is there a need for them?

Reply:
Haven't thought about that. I found it a bit annoying when testing against examples that there is always a diff, but then again if you have a strong feeling about this I can also disable telemetry on my machine

Reply:
Maybe we can disable telemetry for all tests for all developers working on the CDKTF instead?

Reply:
Yeah, I have set the `CHECKPOINT_DISABLE` env var now on my system, you are right that is easier